evm_arithmetization/tests/two_to_one_agg.rs

```diff
@@ -0,0 +1,693 @@
+use std::collections::HashMap;
```

Comment:
General comment for this testing file:

- I'd rather not see any "caching" mechanism by default. These tests shouldn't be heavy enough to justify such behavior, and they wouldn't be run in CI anyway. When debugging things, one can locally add this as it is just about serializing / deserializing some struct.
- related to the previous comment and CI, we should add `#[ignore]` on all these.
- we can probably remove all these `log::info!("Meta Stage 0:  Setup");` which don't bring much info
- To reduce overhead, I think we could encompass all 4 tests together, so that they share the same `AllRecursiveCircuits` / base `block_proofs` to combine. This should reduce most of the overhead on `weaker` machines.

Reply:
Okay, so I will do the following:

- [ ] remove logging
- [ ] remove caching mechanism
- [ ] combine 4 tests into one.
- [ ] add `#[ignore]` to resulting test.